specs/results/koiosapi-guild.yaml

```diff
@@ -974,9 +974,11 @@ paths:
       summary: Pool Relays
       description: A list of registered relays for all currently registered/retiring (not retired) pools
   /pool_metadata: #RPC
-    get:
+    post:
       tags:
         - Pool
+      requestBody:
+        $ref: "#/components/requestBodies/pool_ids"
```

Comment:
This will make pool ID mandatory, we might need to add a duplicate definition for optional pool ID

Reply:
Good catch, done now